### ISSUE TITLE
Rearrange added cards

### ISSUE THREAD

**Issue by usingapc** [dwainscheeren/dwains-lovelace-dashboard#570]
Hello.  Is there a way to rearrange Lovelace cards that are manually added to a room?  I can't find a rhyme or reason to how they are auto arranging when adding more than 1 to a room now.  Sometimes the are arranged in the order I add them, other times they are not.  When I manually add a card the move icon is not available like other automatically added have.

**Comment by tethlah** [comment]
Yes please, it would be great to add a sort order option or something to added cards.  For some reason my chips card is at the bottom of the page on only one area.  I tired removing all cards and recreating them in the order I wanted them displayed and that didn’t work either.  I even tried renaming the actual files with numbers at the start and that changed the order but not to what I wanted lol

**Comment by prjcarr** [comment]
Adding my support for this one. Is there any way to sort manually added yaml cards, even doing it in the config files?

**Comment by cletus5** [comment]
Yes, it will be a must to have, sincerelly I could't any logic in the arrange of cards

**Comment by mwLabs-eu** [comment]
Would be a gread feature!

**Comment by prjcarr** [comment]
Has anyone been able to find a workaround for this? I tried renaming some of the files, but then the entities disappeared completely. How is the sort order determined?

**Comment by adamgerges** [comment]
Any ideas on a workarounds yet?

**Comment by runphp-john** [comment]
Im not sure if this helps or hinders, but Id like to politely request Dev to consider adding this as gratefully received feature.


Also your Dashboard looks the dogs danglies on my Sony Tap20 in the kitchen, you've brought life back to an overpaid for, under used piece of old ass tech! :D

**Comment by Wickedy123** [comment]
Not tested this exhaustively, but it appears this can be achieved by renaming the files created in cards\areas\<your area name>

For example, my files were originally named as follows to give the following ordering:

![image](https://user-images.githubusercontent.com/46416655/235377482-8939f105-685c-43f4-b711-5df89b11538e.png)

![image](https://user-images.githubusercontent.com/46416655/235377505-84a33955-f78a-4feb-bae3-fdffb7567d6c.png)

Then, renaming as follows changes the order:

![image](https://user-images.githubusercontent.com/46416655/235377519-56e970bb-188c-4275-847d-5d4b311e9259.png)

![image](https://user-images.githubusercontent.com/46416655/235377536-bfd80710-fd09-4437-abf8-00c3532573cf.png)

**Comment by douginoz** [comment]
> Not tested this exhaustively, but it appears this can be achieved by renaming the files created in cards\areas<your area name>
> 
> For example, my files were originally named as follows to give the following ordering:
>
> Then, renaming as follows changes the order:
> 
I have 2 lovelace cards I added for a room.  They are displayed as the Xiamoi card first, then the frigate card.  But the filenames are:
- custom:frigate-card.yaml
- custom:xiamoi-vacuum-map-card.yaml

So renaming them wouldn't help as the filenames are already in the order I'd like to see them displayed in.

**Comment by tethlah** [comment]
That and with some of the filenames using illegal characters you can't edit the filename without jumping through some hoops.

**Comment by albotroz** [comment]
Nothing new on this topic?
I have 2 cameras and in my dashboard (Frigate HASS Cards) 2 cards and 1 event (screens) each.
So:
2 cameras, 2 cards, 2 events for people

The livestreams of the cameras are displayed in the left corner when viewed via desktop and the events on the right. This makes no sense at all.

On smartphones, on the other hand, the view is such that the live streams are displayed first and then the events. Everything below each other, which looks good.

This is how it is: 
![1](https://github.com/dwainscheeren/dwains-lovelace-dashboard/assets/67508404/1808d5b2-0268-4e86-a613-28e047c97902)


That's how I would like it to be:
![2](https://github.com/dwainscheeren/dwains-lovelace-dashboard/assets/67508404/2357849e-ba6a-40ee-bc7e-22c1aa424fd4)

**Comment by dwainscheeren** [comment]
Thanks for reporting this.

Dwains Dashboard 3.9.0 has now been released, and the source files for DD3 are available in this repository.

DD3 is no longer under active feature development and will slowly be phased out. Active development continues in Dwains Dashboard Next:
https://github.com/dwainscheeren/dwains-dashboard-next

At the time of writing, Dwains Dashboard Next is still in alpha.

If this issue is a feature request, please check whether it fits Dwains Dashboard Next instead.

If this issue is a bug, please retest with DD3 version 3.9.0. If the bug is still reproducible there, please open a new issue with:

- Home Assistant version
- Dwains Dashboard version
- browser and device
- browser console errors
- clear reproduction steps

For more context, see #906.

For that reason, this older issue is being closed as part of the DD3 issue tracker cleanup.
